packages/ng/callout/callout-disclosure/callout-disclosure.component.ts

```diff
@@ -44,4 +44,14 @@ export class CalloutDisclosureComponent {
 			this.icon = icon;
 		}
 	}
+
+	@Input({ transform: booleanAttribute }) open = false;
+
+	@Output() openChange = new EventEmitter<boolean>();
+
+	public onToggle(event: Event) {
+		if (event.target instanceof HTMLDetailsElement) {
```

Comment:
Why is this check needed? Do we have something that could trigger this without being from the details element?

Reply:
I prefer to check the typing of the target rather than casting it.

This way of checking type is already present in the code (I can reverse condition) : https://github.com/LuccaSA/lucca-front/blob/rc/packages/ng/multi-select/panel/panel.component.ts#L50